integration-cli/docker_cli_exec_test.go

```diff
@@ -593,25 +497,20 @@ func (s *DockerSuite) TestExecWithImageUser(c *check.C) {
 		RUN echo 'dockerio:x:1001:1001::/bin:/bin/false' >> /etc/passwd
 		USER dockerio`,
 		true)
-	if err != nil {
-		c.Fatalf("Could not build image %s: %v", name, err)
-	}
+	c.Assert(err, checker.IsNil)
 
 	dockerCmd(c, "run", "-d", "--name", "dockerioexec", name, "top")
 
 	out, _ := dockerCmd(c, "exec", "dockerioexec", "whoami")
-	if !strings.Contains(out, "dockerio") {
-		c.Fatalf("exec with user by id expected dockerio user got %s", out)
-	}
+	c.Assert(out, checker.Contains, "dockerio")
 }
 
 func (s *DockerSuite) TestExecOnReadonlyContainer(c *check.C) {
 	// --read-only + userns has remount issues
 	testRequires(c, DaemonIsLinux, NotUserNamespace)
 	dockerCmd(c, "run", "-d", "--read-only", "--name", "parent", "busybox", "top")
-	if _, status := dockerCmd(c, "exec", "parent", "true"); status != 0 {
-		c.Fatalf("exec into a read-only container failed with exit status %d", status)
-	}
+	_, status := dockerCmd(c, "exec", "parent", "true")
```

Comment:
Same here, you can drop the assert, and remove the left part :

``go
dockerCmd(c, "exec", "parent", "true")
``